go/exorelay/exorelay_test.go

```diff
@@ -130,6 +167,56 @@ func FeatureContext(s *godog.Suite) {
 		}
 		return nil
 	})
+
+	s.Step(`^I register a handler for the "([^"]*)" message$`, func(messageName string) error {
+		mapping := map[string]func(map[string]interface{}){}
+		mapping[messageName] = func(payload map[string]interface{}) {
+			handlerCalled = true
```

Comment:
handlerCalled is not called for all handlers. It is set for this one specific handler though. We could make it a map, but I don't think that would be worth it.